frontend/src/Components/LegalNotice/LegalNotice.tsx

```diff
@@ -2,116 +2,190 @@ import classes from './LegalNotice.module.css';
 
 export const LegalNotice = () => (
   <div class={classes.legalNotice}>
-    <h1>Impressum</h1>
+    <h1>Company and Contact Information</h1>
```

Comment:
Is this section still needed?

Reply:
Yes it is, every page must have a "Impressum" reachable with no more than 2 clicks